plugin-server/src/utils/db/db.ts

```diff
@@ -1442,20 +1486,27 @@ export class DB {
 
     // Hook (EE)
 
-    public async fetchRelevantRestHooks(
-        teamId: Hook['team_id'],
-        event: Hook['event'],
-        resourceId: Hook['resource_id']
-    ): Promise<Hook[]> {
-        const filterByResource = resourceId !== null
-        const { rows } = await this.postgresQuery<Hook>(
-            `
-            SELECT * FROM ee_hook
-            WHERE team_id = $1 AND event = $2 ${filterByResource ? 'AND resource_id = $3' : ''}`,
-            filterByResource ? [teamId, event, resourceId] : [teamId, event],
-            'fetchRelevantRestHooks'
-        )
-        return rows
+    private async fetchActionRestHooks(resource_id?: Hook['resource_id']): Promise<Hook[]> {
+        try {
+            const { rows } = await this.postgresQuery<Hook>(
+                `
+                SELECT *
+                FROM ee_hook
+                WHERE event = 'action_performed'
+                ${resource_id !== undefined ? 'AND resource_id = $1' : ''}
+                `,
+                resource_id !== undefined ? [resource_id] : [],
+                'fetchActionRestHooks'
+            )
+            return rows
+        } catch (err) {
+            // On FOSS this table does not exist - ignore errors
+            if (err.message.includes('relation "ee_hook" does not exist')) {
```

Comment:
Did you test this locally? Alternatively, can we have an integration test for this?

Reply:
There's a unit test here: https://github.com/PostHog/posthog/pull/10414/files#diff-8d6ee36ecb2a27d3237a685e842e0e143be815caf84da31dbe5e60985dac90a9R224-R239

Reply:
Ah nice, I searched for `DROP TABLE` or lack of `CREATE TABLE` but missed the `ALTER TABLE` 😅 (BTW I'd argue this is an integration test, since it relies on the behavior of a live Postgres server, but that's a totally different topic)